modules/appeals_api/app/controllers/appeals_api/notice_of_disagreements/v0/notice_of_disagreements_controller.rb

```diff
@@ -30,13 +33,40 @@ class NoticeOfDisagreementsController < AppealsApi::V2::DecisionReviews::NoticeO
       ]
     }.freeze
 
+    def show
+      nod = AppealsApi::NoticeOfDisagreement.select(ALLOWED_COLUMNS).find(params[:id])
```

Comment:
I was curious what impact the `.select(ALLOWED_COLUMNS)` had on the GET response, so I pulled your branch and played around with it by modifying `ALLOWED_COLUMNS`. Also, it looks like the responses before and after this change match. Looks good to me!